portfolio/src/main/webapp/script.js

```diff
@@ -156,31 +159,50 @@ function getComments() {
 }
 
 /**
- * Create element that represents a comment.
+ * Format timestamp for display in comment section.
  */
-function createCommentElement(comment) {
-  const commentElement = document.createElement('div');
-
-  const nameElement = document.createElement('div');
-  nameElement.innerText = comment.name;
-
-  const textElement = document.createElement('div');
-  textElement.innerText = comment.text;
-
-  const timeElement = document.createElement('div');
-  const timestampString = new Date(comment.timestamp).toString();
-  timeElement.innerText = timestampString;
-
-  const scoreElement = document.createElement('div');
-  scoreElement.innerText = 'Sentiment score (Scale from -1 to 1): ' +
-      comment.sentimentScore.toString();
+function parseTimestamp(timestamp) {
+  const date = new Date(timestamp);
+  return date.toLocaleString();
+}
 
-  commentElement.appendChild(nameElement);
-  commentElement.appendChild(textElement);
-  commentElement.appendChild(timeElement);
-  commentElement.appendChild(scoreElement);
+/**
+ * Get name of icon that represents sentiment score.
+ */
+function getIcon(sentimentScore) {
+  /* Conditions are ordered. */
+  if (sentimentScore >= 0.5) {
```

Comment:
Since you have returns the else is not needed. so something like

```
if (sentimentScore >= 0.5) return '...';
if (sentimentScore >= 0) return '...';
if (sentimentScore >= -0.5) return '...';
return '...'
```